### PR TITLE
Fix line split in CI workflow file

### PR DESCRIPTION
**Pull request by Jasagredo**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -103,7 +103,8 @@ jobs:
 
     - name: Record dependencies
       id: record-deps
-      run: cabal build all --dry-run
+      run: |
+        cabal build all --dry-run
```
Comment (amesgen) [inline .github/workflows/build.yml:107]: Why did this not make CI fail?

https://github.com/input-output-hk/ouroboros-network/actions/runs/4084241519/jobs/7040712757#step:11:13

**Review by amesgen — APPROVED** [review]
Thanks, I guess you did note this just as a side effect, or does this actually fix an issue?

Might be a nice addition to https://noyaml.com

**Comment by Jasagredo** [comment]
> Thanks, I guess you did note this just as a side effect, or does this actually fix an issue?
> 
> 
> 
> Might be a nice addition to https://noyaml.com

I actually found it while thinking what could be causing that CI error on Arnaud's branch, and wanting to check the dependencies file if possible 😄

File: .github/workflows/build.yml (hunk 1 above)
Comment (Jasagredo) [reply .github/workflows/build.yml:107]: Tbh I have no idea. Perhaps Cabal doesn't end with an exit code that is recognized as failing in this case?

**Comment by Jasagredo** [comment]
bors retry

**Pull request merged**